tests/generic_config_updater/files/patch_sorter_test_success.json

```diff
@@ -2811,5 +2811,88 @@
                 }
             ]
         ]
+    },
+    "ADDING_LOOPBACK0_VRF_NAME__DELETES_LOOPBACK0_AND_IPS_DOES_NOT_AFFECT_OTHER_TABLES": {
```

Comment:
This test case was failing because at some point the patch-sorter
- tries to delete the whole config move which is rejected using which was generated using: DeleteInsteadOfReplaceMoveExtender 
- tries to delete the referrers to the whole config : DeleteRefsMoveExtender 
- why trying to remove referrers of whole config, it actually tries to delete referrers to PORT table i.e. CABLE_LENGTH.

Reply:
resolve the issue